i18n/de.toml

```diff
@@ -66,24 +66,20 @@ other = "Kapitel inkl. Unterseiten drucken"
 
 # Community
 [community]
-other = "community"
+other = "Community"
 [community_join]
-other = "Join the"
+other = "Werde Teil der {{ .Site.Title }} community"
```

Comment:
`other = "Werde Teil der {{ .Site.Title }} community"`

`Community` must be uppercase here:

Correct is:

`other = "Werde Teil der {{ .Site.Title }} Community"`